src/views/albums/Album.vue

```diff
@@ -61,27 +61,33 @@
 </template>
 
 <script>
-import { mapState } from "vuex";
+import { mapState, mapActions } from "vuex";
 import AlbumActions from "../../components/AlbumActions";
 import TracksTable from "../../components/TracksTable";
 import AlbumArtists from "../../components/AlbumArtists";
+import { TracksScope } from "@/api/scopes";
 
 export default {
   name: "Album",
   components: { AlbumArtists, TracksTable, AlbumActions },
   metaInfo() {
     return { title: this.album?.title };
   },
+  props: {
+    id: {
+      type: [String],
```

Comment:
Why did you remove the `, Number` here?

Reply:
I was experimenting with these types to reconstruct why I added both. This slipped in by accident.